tests/sentry/middleware/integrations/parsers/test_discord.py

```diff
@@ -130,3 +136,29 @@ def test_control_classes(self):
                 parser.get_response()
                 assert mock_response_from_control.called
                 assert not get_response_from_outbox_creation.called
+
+
+class DiscordTestEndpoint(Endpoint):
+    permission_classes = (AllowAny,)
+
+    def post(self, request):
+        return JsonResponse({"foo": "bar"})
+
+
+urlpatterns = [
+    re_path(
+        r"^extensions/discord/interactions/$",
+        DiscordTestEndpoint.as_view(),
+        name="discord-test-endpoint",
+    ),
+]
+
+
+@control_silo_test
+@override_settings(ROOT_URLCONF=__name__)
+class End2EndTest(APITestCase):
+    def test_discord_request_parser(self):
```

Comment:
I might be missing something, where does this test the discord parser itself? I don't see the discord request parser getting used

Reply:
@mrduncan I re-iterated and cleaned up the tests. 

My intention here was to try to hit the route `/extensions/discord/interactions/` in the control silo mode. Doing so should implicitly invoke the Discord request parser, and at the same time, run the Request and Response objects through the middleware layers.